build.js

```diff
@@ -1,10 +1,27 @@
+const fs = require('fs');
+const path = require('path');
+const Ajv = require('ajv');
```

Comment:
Is the default export a class definition?

Reply:
indeed 
<img width="509" alt="image" src="https://github.com/ar-io/arns-pilot/assets/12800001/90bea10a-1cf9-46fa-9607-329e05d9e5fc">